src/app/content/highlights/components/Card.tsx

```diff
@@ -121,13 +135,14 @@ const Card = (props: CardProps) => {
     className: props.className,
     highlight: props.highlight,
     isFocused: props.isFocused,
+    moveFocusToTheHighlight: props.moveFocusToTheHighlight,
```

Comment:
you don't have to pass this down, you could handle it with `setEditing` on line 160